topi/tests/python/test_topi_sort.py

```diff
@@ -27,6 +27,12 @@ def verify_argsort(axis, is_ascend):
     data_dtype = "float32"
     data = tvm.placeholder(dshape, name="data", dtype=data_dtype)
     np_data = np.random.uniform(size=dshape).astype(data_dtype)
```

Comment:
remove np_data here

Reply:
Do you mean the output `nd.array` of `numpy.random.uniform`?

Reply:
np data is provided in the next few lines, this line is not needed

Reply:
I think `np_data` is needed by the following lines, like L32 and L37, but please correct me if I’m wrong on that

Reply:
Ah, sorry I was a bit confused here. The shuffling here still does not address the problem of ties(two values too close to each other in random.uniform). 

We should instead change this line to directly create arange then broadcast to the shape, then we shuffle each of the broadcasted place. 

Then the problem will be solved because arange won't have ties

Reply:
Apology for the misunderstanding. I update that part you mentioned and the test results as shown below:

```
>>> perm = np.arange(dshape[0] * dshape[1], dtype=data_dtype)
>>> np.random.shuffle(perm)
>>> np_data = perm.reshape(dshape)
>>> np_data
array([[22., 23., 10., 18., 12.],
       [16., 20.,  6., 21.,  1.],
       [24.,  7.,  4., 15., 13.],
       [14.,  5.,  8., 19., 11.],
       [ 9.,  3.,  0.,  2., 17.]], dtype=float32)
```